init-templates/gqlgen.yml.gotmpl

```diff
@@ -92,8 +131,23 @@ models:
       - github.com/99designs/gqlgen/graphql.Int
       - github.com/99designs/gqlgen/graphql.Int64
       - github.com/99designs/gqlgen/graphql.Int32
+  UUID:
+    model:
+      - github.com/99designs/gqlgen/graphql.UUID
```

Comment:
hmmmm....

Reply:
I think this deserves a comment as well. The graphql.UUID uses "github.com/google/uuid", but there's a fair number of people who want to use `github.com/gofrs/uuid` or `github.com/jackc/pgx-gofrs-uuid` so we should have some comment here referencing documentation like the pgx project has for https://github.com/jackc/pgx/wiki/UUID-Support